### PR TITLE
MINOR: Add metrics in operations doc for KIP-429 / KIP-467

### PR DESCRIPTION
**Pull request by guozhangwang**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by guozhangwang** [comment]
@tuvtran @ableegoldman @vvcephei

File: docs/ops.html [hunk 1]
```diff
@@ -1459,11 +1479,81 @@ <h5><a id="consumer_group_monitoring" href="#consumer_group_monitoring">Consumer
...
+        <td>The number of seconds since the last rebalance event</td>
+        <td>kafka.consumer:type=consumer-coordinator-metrics,client-id=([-.\w]+)</td>
+      </tr>
       <tr>
         <td>last-heartbeat-seconds-ago</td>
         <td>The number of seconds since the last controller heartbeat</td>
         <td>kafka.consumer:type=consumer-coordinator-metrics,client-id=([-.\w]+)</td>
       </tr>
+      <tr>
+        <td>partitions-revoked-time-avg</td>
```
Comment (ableegoldman) [inline docs/ops.html]: Is this metric partitions-revoked-**time**-avg or partition-revoked-**latency**-avg ? (The PR description indicates it's the latter)

**Review by ableegoldman — APPROVED** [review]
The 429 metrics look good, modulo a question about the callback metrics names

**Comment by ableegoldman** [comment]
@guozhangwang Does this include the additional metric that @vvcephei added?

File: docs/ops.html (hunk 1 above)
Comment (guozhangwang) [reply docs/ops.html]: Good catch! It should be `partition-revoked-latency-avg` and ditto others -- I was confused by the KIP wiki page itself.

**Comment by guozhangwang** [comment]
> @guozhangwang Does this include the additional metric that @vvcephei added?

Good catch again! :) Will add.

EDIT: actually mvn, I've already added it.

**Comment by ableegoldman** [comment]
> EDIT: actually mvn, I've already added it.

Ah ok thanks for checking, wasn't sure which one that was. LGTM!

**Review by tuvtran — APPROVED** [review]
The metrics in KIP 467 LGTM! Thanks @guozhangwang!

**Review by mjsax — APPROVED** [review]
LGTM.

**Comment by guozhangwang** [comment]
Cherry-picked to 2.4.

**Pull request merged**